### PR TITLE
zforce: add page

### PR DESCRIPTION
**Pull request by gutjuri**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

This command has no options, so its tldr page is rather short.

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

Wrapping extensions in quotes is perfect, will try to add it style guide as we don't have a set convention for it now.

Kinda curious if non-gzip files won't be modified, why should we place them in the placeholder?

**Comment by gutjuri** [comment]
> Kinda curious if non-gzip files won't be modified, why should we place them in the placeholder?

I have placed one non-gzipped file to demonstrate that `zforce` can deal with such files and not accidentally put a '.gz' extension on them.

File: pages/linux/zforce.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# zforce
+
+> Add a '.gz' extension on all supplied gzip files.
```
Comment (kbdharun) [inline pages/linux/zforce.md:3]:
```suggestion
> Add a GZ extension on all supplied GZip files.
```
For page descriptions, we could go with capitalized extensions/names. (We can leave the examples using quotes to wrap extensions as it is)

File: pages/linux/zforce.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+> More information: <https://manned.org/zforce.1>.
+
+- Add a '.gz' extension to the supplied gzip files, leave the suppied non-gzip files as-is:
+
+`zforce {{path/to/gzipped1}} {{path/to/gzipped2}} {{path/to/non/gzipped}}`
```
Comment (kbdharun) [inline pages/linux/zforce.md:8]:
```suggestion
- Add a '.gz' extension to the supplied GZip files (leave the supplied non-gzip files as-is):

`zforce {{path/to/archive1.gz path/to/archive2.gz ...}}`
```

**Review by kbdharun — APPROVED** [review]

**Comment by kbdharun** [comment]
> Why do you think single quotes are better for extensions?

Both quotes/backticks work fine for extensions, we have a couple of pages using it. We can update them to backticks now that we have formalized it as default in the style guide (in the merged PR).

File: pages/linux/zforce.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# zforce
+
+> Add a GZ extension on all supplied GZip files.
+> More information: <https://manned.org/zforce.1>.
+
+- Add a '.gz' extension to the supplied GZip files (Note: Non-GZip files are ignored):
```
Comment (kbdharun) [inline pages/linux/zforce.md:6]:
```suggestion
- Add a `.gz` extension to the supplied GZip files (Note: Non-GZip files are ignored):
```

File: pages/linux/zforce.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# zforce
+
+> Add a GZ extension on all supplied GZip files.
+> More information: <https://manned.org/zforce.1>.
```
Comment (acuteenvy) [inline pages/linux/zforce.md:4]:
```suggestion
> Add a `.gz` extension to files compressed using `gzip`.
> More information: <https://manned.org/zforce>.
```

File: pages/linux/zforce.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+
+- Add a `.gz` extension to the supplied GZip files (Note: Non-GZip files are ignored):
```
Comment (acuteenvy) [inline pages/linux/zforce.md:6]:
```suggestion
- Add a `.gz` extension to the supplied Gzip files (Note: other files are ignored):
```

File: pages/linux/zforce.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
...
+
+`zforce {{path/to/archive1.gz path/to/archive2.gz ...}}`
```
Comment (acuteenvy) [inline pages/linux/zforce.md:8]: This won't do anything, will it? The files already end with `.gz`.
```suggestion
`zforce {{path/to/file1 path/to/file2 ..}}`
```

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**